test/stored_queries/test_ncbi_tax.py

```diff
@@ -59,18 +71,18 @@ def setUpClass(cls):
             {'_from': 'ws_object_version/1:1:2', '_to': 'ncbi_taxon/1', 'assigned_by': 'assn2'},
             {'_from': 'ws_object_version/2:1:1', '_to': 'ncbi_taxon/1', 'assigned_by': 'assn2'},
         ]
-        ws_docs = [{'_key': '1', 'is_public': True}, {'_key': '2', 'is_public': False}]
+        ws_docs = [{'_key': '1', 'is_public': True}, {'_key': '1', 'is_public': False}]
```

Comment:
Not sure if this change was intentional? If so, what's the purpose?

Reply:
Good catch thanks